app/helpers/application_helper.rb

```diff
@@ -385,19 +385,22 @@ def override_backlink
     return link
   end
 
-  def display_carrier_logo(plan, options = {:width => 50})
+  def carrier_logo(plan)
     return "" if !plan.carrier_profile.legal_name.extract_value.present?
-    carrier_name = plan.carrier_profile.legal_name.extract_value
+    issuer_hios_id = plan.hios_id[0..4].extract_value
```

Comment:
are all plan hios ID's going to be using this range ?

Reply:
Every plan hios_id is using [0..4] range, For generating uniqueness based of the first 5 digits of hios_id.

Reply:
DC UHC has 3 different Logos but the legal name is same, so we need to identify the logo based on some unique identifier. We consider hios_id's first 5 digit are unique for all these and loading logo accordingly.